File: DATN_Server/app/Models/Teacher.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Teacher extends Model
{
    use HasFactory;

    protected $fillable = [
        'teacher_id',
        'teacher_name',
        'email',
        'birthday',
        'phone',
        'topic',
        'avt',
        'title',
        'major_id'
    ];

    protected $primaryKey = 'teacher_id';
}

File: DATN_Mobile/app/src/main/java/com/example/qldatn/models/Teacher.kt
package com.example.qldatn.models

import java.io.Serializable

class Teacher(
              val teacher_id:Int,
              val teacher_name:String,
              val email:String,
              val birthday:String,
              val phone:String,
              val topic:String,
              val platforms:String,
              val quota:Int,
              val available_quota:Int,
              val major:String,
              val avt:String,
              val major_id:Int) : Serializable {
}